docs/wiki/_Sidebar.md

```diff
@@ -9,6 +9,8 @@
 - [Testing](./Testing)
   - [Design](./TestingDesign)
   - [Usage](./TestingUsage)
+- [Utilities](./Utilities.md)
+  - [Bicep to ARM conversion script](./UtilitiesConversionScript.md)
```

Comment:
Remove child. The sidebar should only display the most important navigation points. Listing the individual tools would get a bit out of hand

Reply:
done